### PR TITLE
8298876: Swing applications do not get repainted coming out of sleep on Windows 10

### PR DESCRIPTION
**Pull request by prsadhuk**
This is a revert of commit https://github.com/openjdk/jdk/commit/415bf44191632cd8dbcc158c0ff0992c0b61c3ba
which was a fix for 8275715: D3D pipeline processes multiple PaintEvent at initial drawing 
as it is causing some repaint issue and also since the regression test is known to fail in some systems and is currently problemlisted which suggests the fix was half-baked.

Raised a new issue for the same https://bugs.openjdk.org/browse/JDK-8299436

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8298876](https://bugs.openjdk.org/browse/JDK-8298876): Swing applications do not get repainted coming out of sleep on Windows 10

### Reviewers
 * [Andy Goryachev](https://openjdk.org/census#angorya) (@andy-goryachev-oracle - no project role)
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11808/head:pull/11808` \
`$ git checkout pull/11808`

Update a local copy of the PR: \
`$ git checkout pull/11808` \
`$ git pull https://git.openjdk.org/jdk pull/11808/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11808`

View PR using the GUI difftool: \
`$ git pr show -t 11808`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11808.diff">https://git.openjdk.org/jdk/pull/11808.diff</a>

</details>

### CODE REVIEW

**Comment by mrserb** [comment]
I suggest getting confirmation from Andy that this change will actually fix the problem.

**Review by andy-goryachev-oracle — APPROVED** [review]
The issue disappears with this change.  Many thanks!

**Review by prrace — APPROVED** [review]

**Comment by mrserb** [comment]
Since this is just a revert what about fixing it in jdk20?

File: test/jdk/sun/java2d/DirectX/MultiPaintEventTest/MultiPaintEventTest.java [hunk 1]
```diff
@@ -1,86 +0,0 @@
-/*
```
Comment (mrserb) [inline test/jdk/sun/java2d/DirectX/MultiPaintEventTest/MultiPaintEventTest.java:1]: The test should be removed from the problem list as well.

**Comment by prrace** [comment]
> Since this is just a revert what about fixing it in jdk20?

If it had been introduced in 20 we likely would but it was introduced in 19.

**Pull request closed without merging**